src/java.compiler/share/classes/javax/lang/model/util/Types.java

```diff
@@ -263,7 +263,9 @@ public interface Types {
      *
      * @param componentType  the component type
      * @throws IllegalArgumentException if the component type is not valid for
-     *          an array, including executable, package, module, and wildcard types
+     *          an array. All valid types are {@linkplain ReferenceType
+     *          reference types} or {@linkplain PrimitiveType primitive types}.
+     *          Invalid types include null, executable, package, module, and wildcard types.
```

Comment:
Should it be somehow more clear that it's not `null`, but the __Null Type__. Maybe you can link to it as follows?

    {@linkplain NullType null}

Reply:
> Should it be somehow more clear that it's not `null`, but the **Null Type**. Maybe you can link to it as follows?
> 
> ```
> {@linkplain NullType null}
> ```

Good suggestion; implemented.